styles/helpers/menu-slide.css

```diff
@@ -16,8 +16,22 @@ title: Mobile Menu
   </div>
 </div>
 ```
+```example:html
+<div class="menu-slide left">
```

Comment:
I think it's too risky to add the modifier as just `left`.  What if we add a helper class down the road that is just `left`?